framework/src/application/node/node.js

```diff
@@ -330,6 +331,7 @@ module.exports = class Node {
 	}
 
 	async cleanup(error) {
+		await this.transactionPool.stop();
```

Comment:
We do have a convention of `bootstrap`  and `cleanup` method in every module? Should not we follow same here instead of `start/stop`?

Reply:
I think for p2p, it's start and stop. Also, for p2p and transaction pool, it's running the interval and not just bootstrapping, so it's better to call start than bootstrap IMO